src/styles/blocks/page.scss

```diff
@@ -0,0 +1,9 @@
+body {
+  margin: 0;
+  width: 100vw;
```

Comment:
Using 'width: 100vw;' can cause horizontal scrolling issues on some browsers due to the presence of scrollbars. Consider using 'width: 100%;' instead.